graph/auxiliary/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ost-anchor",
+  "license": "UNLICENSED",
```

Comment:
Is there a reason to go without license here?

Reply:
Added